sympy/functions/elementary/complexes.py

```diff
@@ -754,6 +852,35 @@ def _eval_is_algebraic(self):
 class transpose(Function):
     """
     Linear map transposition.
+
+    Examples
+    ========
+
+    >>> from sympy.functions import transpose
+    >>> from sympy.matrices import MatrixSymbol
+    >>> A = MatrixSymbol('A', 25, 9)
+    >>> transpose(A)
+    A.T
+    >>> B = MatrixSymbol('B', 9, 22)
+    >>> transpose(B)
+    B.T
+    >>> transpose(A*B)
+    B.T*A.T
+
+    Parameters
+    ==========
+
+    arg : MatrixSymbol
+        Symbolic representation of a Matrix Object.
+
+    Returns
+    =======
+
+    value : MatrixExpr
+        MatrixExpr is the superclass for matric expressions.
+        Represent abstract matrices, linear transformations
+        represented within a particular basis.
```

Comment:
I don't think that this is a place to explain MatrixExpr here.
Maybe you should explain something like 
`This returns an according transpose object if the object represents any form of linear map. A common example would be a matrix with its usual transpose.`

Reply:
Made some changes.